### DISCUSSION TITLE
[Feature Request] Lack of clarity on why some components are not included in lion

### DISCUSSION THREAD

**Discussion by jorenbroekema** [ing-bank/lion#711] [General]
## Problem

People will often expect certain UI components to be inside lion. If they aren't, the default assumption is probably that it's missing, we forgot, or we haven't gotten to it yet. 

Often, it's a conscious decision however. Because we've decided that a certain component is best left to the extension design system to build from scratch, and there's no useful base-layer or white-label abstraction for us to put in lion for it. An example could be a grid layout component, or a table. It's hard for us to abstract such a component to a base layer.

To put it in analogies, we create the basis of a delicious dish, but you need to put your own sauce on top of it. But for some dishes, the sauce is the entire dish. Or it's just a soup. There's no meaningful base recipee for us to supply.

## Proposal

A clear list in our main README.md of components that we consciously decided not to include in lion, with a short rationale on why (e.g. because there is no meaningful abstraction that isn't sufficiently white labelled), and optionally a list of resources to go to for inspiration on creating these components from scratch.

**Comment by petecarapetyan** [comment]
Perhaps because I had (childishly) always looked for WCs as the solution to any problem, and because of my familiarity with Vaadin Grid, I already had my options “set” in this direction.
So the absence of a grid in Lion set off alarm bells in my brain. I started looking at other component kits - “Where is your grid?”
My reason for bringing up this point might seem silly - but I only looked under lists of WCs. Whereas your guidance on this might never be found in such a list.  (You can’t find something not offered)
So the question here is how to place your guidance where people will look?
Or another opposite crazy is “I would consider Lion but for it’s lack of grid”
Only partially joking when I jest: “Maybe a WC that only contains external CSS options to import” just so you can get it where people look.

**Comment by petecarapetyan** [comment]
Here is some guidance that also includes javascript as part of the solution. 
https://www.w3.org/TR/wai-aria-practices/examples/grid/dataGrids.html

Including it here only to further along my jest from above that an almost fake WC as container of externally supplied CSS might offer a direction. Even if this direction was to develop the WC yourself, Mr. Developer?
